### PR TITLE
New signing key for packages

### PR DESCRIPTION
**Pull request by alephnull**
This PR will contain separate commits for the various components as they are implemented. This commit handles the gateway. Commits are pending for:

- dashboard
- pump

The Ubuntu package origin verification is part of this. 

The merging of this PR into master will need some co-ordination between us.

### CODE REVIEW

**Review by marksou — APPROVED** [review]
Sorted a couple of typos. Good to go from my point of view.

**Comment by marksou** [comment]
@excieve  Can we review this so we can release as part of 2.9.4?

File: tyk-docs/content/getting-started/installation/with-tyk-on-premises/redhat-rhel-centos/gateway.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ baseurl=https://packagecloud.io/tyk/tyk-gateway/el/7/$basearch
 repo_gpgcheck=1
 gpgcheck=1
 enabled=1
-gpgkey=http://keyserver.tyk.io/tyk.io.rpm.signing.key
+gpgkey=http://keyserver.tyk.io/tyk.io.rpm.signing.key.2020
```
Comment (excieve) [inline tyk-docs/content/getting-started/installation/with-tyk-on-premises/redhat-rhel-centos/gateway.md]: Might want an https here if possible.

File: tyk-docs/content/getting-started/installation/with-tyk-on-premises/on-ubuntu/gateway.md [hunk 2]
```diff
@@ -39,12 +39,14 @@ Since our repositories are installed via HTTPS, you will need to make sure APT s
 sudo apt-get install -y apt-transport-https 
 ```
 
-Now lets add the required repos and update again (notice the `-a` flag in the second Tyk commands - this is important!):
+Create a file `/etc/apt/sources.list.d/tyk_tyk-gateway.list` with the following contents:
 ```{.copyWrapper}
-echo "deb https://packagecloud.io/tyk/tyk-gateway/ubuntu/ trusty main" | sudo tee /etc/apt/sources.list.d/tyk_tyk-gateway.list
-
-echo "deb-src https://packagecloud.io/tyk/tyk-gateway/ubuntu/ trusty main" | sudo tee -a /etc/apt/sources.list.d/tyk_tyk-gateway.list
+deb https://packagecloud.io/tyk/tyk-gateway/ubuntu/ trusty main
```
Comment (excieve) [inline tyk-docs/content/getting-started/installation/with-tyk-on-premises/on-ubuntu/gateway.md]: @marksou Should we change `trusty` (14.04) to something more recent here throughout the document?

**Review by excieve — COMMENTED** [review]
LGTM in general, see notes please.

File: tyk-docs/content/getting-started/installation/with-tyk-on-premises/redhat-rhel-centos/gateway.md (hunk 1 above)
Comment (alephnull) [reply tyk-docs/content/getting-started/installation/with-tyk-on-premises/redhat-rhel-centos/gateway.md]: done

File: tyk-docs/content/getting-started/installation/with-tyk-on-premises/on-ubuntu/gateway.md (hunk 2 above)
Comment (marksou) [reply tyk-docs/content/getting-started/installation/with-tyk-on-premises/on-ubuntu/gateway.md]: @excieve is `xenial main` ok?

**Comment by alephnull** [comment]
@marksou This is ready to be released now.

File: tyk-docs/content/getting-started/installation/with-tyk-on-premises/on-ubuntu/gateway.md (hunk 2 above)
Comment (marksou) [reply tyk-docs/content/getting-started/installation/with-tyk-on-premises/on-ubuntu/gateway.md]: Updated to `bionic`

**Pull request merged**